src/components/Calculator.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] OPTIONAL
You have **disabled ESLint linter check** on your code. Please enable **all** linter check on your code to ensure that your code is robust code and clean.

Reply:
@rloterh 
Thank you for your feedback. :+1: